packages/typespec-ts/test/modularUnit/modelsGenerator.spec.ts

```diff
@@ -1712,246 +1717,6 @@ describe("inheritance & polymorphism", () => {
         true
       );
     });
-
-    it.skip("extensible enums with string literals being used in regular headers", async () => {
```

Comment:
are they removed because we added in enumUnion.spec.ts ?

Reply:
yes, all cases are moved to a seperated file